timeless/poster/tasks.py

```diff
@@ -59,6 +59,23 @@ def sync_customers():
         )
 
 
+@shared_task
+def sync_locations():
+    """
+    Periodic task for fetching and saving location from Poster
+    """
+    for poster_location in __poster_api().locations():
+        location = DB.session(Location).query.filter_by(
+            name=poster_location["name"],
+            code=poster_location["code"]
+        ).first()
+        merge_data(
```

Comment:
@vladarefiev maybe I didn't understand, but shouldn't this method call actually be ``merge_with_poster`` from restaurants models?

Reply:
@amihaiemil Now, why do you think so? We are merging poster location with Location instance. So, it should be `Location.merge_with_poster(...)`

Reply:
@vladarefiev yes, exactly, ``Location.merge_with_poster(...)`` should be called -- then why is it ``merge_data`` now?

Reply:
@amihaiemil `merge_data` is utility function as I can see, it was implemented before me. but I guess actually it was created by complexity reasons, I bet codeclimate was yelling

Reply:
@vladarefiev I see. Then let's leave it like that.